src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -111,8 +126,7 @@
       </div>
     </div>
     <kd-user-help>
-      The specified labels will be applied to the created Replica Set, Service (if any) and Pods.
-      Common labels include release, environment, tier, partition and track.
+      The specified labels will be applied to the created Replica Set, Service (if any) and Pods. Common labels include release, environment, tier, partition and track.
```

Comment:
Hmm... Why did you modify this? :) Please revert. Also, try to stay with 100 character limited lines.